extensions/amp-animation/0.1/web-animations.js

```diff
@@ -126,7 +127,7 @@ export class WebAnimationRunner {
       // Apply vars.
       if (request.vars) {
         for (const k in request.vars) {
-          request.target.style.setProperty(k, String(request.vars[k]));
+          setImportantStyles(request.target, {[k]: String(request.vars[k])});
```

Comment:
This should be `setStyles` to preserve current behavior, right?

Reply:
right. good catch. Done.